libs/shared/console-shared/src/lib/job-general-setting/job-general-setting.tsx

```diff
@@ -0,0 +1,122 @@
+import { BuildModeEnum } from 'qovery-typescript-axios'
+import { Controller, useFormContext } from 'react-hook-form'
+import { IconEnum, ServiceTypeEnum } from '@qovery/shared/enums'
+import { JobGeneralData, OrganizationEntity } from '@qovery/shared/interfaces'
+import { BlockContent, Icon, InputSelect, InputText } from '@qovery/shared/ui'
+import CreateGeneralGitApplication from '../create-general-git-application/create-general-git-application'
+import GeneralContainerSettings from '../general-container-settings/ui/general-container-settings'
+import EditGitRepositorySettingsFeature from '../git-repository-settings/edit-git-repository-settings-feature/edit-git-repository-settings-feature'
+
+export interface JobGeneralSettingProps {
+  organization?: OrganizationEntity
+  jobType: 'cron' | 'lifecycle'
+  isEdition?: boolean
+}
+
+export function JobGeneralSetting(props: JobGeneralSettingProps) {
+  const { control, getValues, watch } = useFormContext<JobGeneralData>()
+  watch('serviceType')
```

Comment:
Is not useful

Reply:
It is, if you remove it the view is not rendered with just the serviceType changing. We met this case previously in another funnel flow